services/spar/src/Spar/Scim/User.hs

```diff
@@ -271,7 +271,8 @@ validateScimUser' errloc midp richInfoLimit user = do
     either err pure $ Brig.mkUserName (Scim.displayName user) veid
   richInfo <- validateRichInfo (Scim.extra user ^. ST.sueRichInfo)
   let active = Scim.active user
-  pure $ ST.ValidScimUser veid handl uname richInfo (maybe True Scim.unScimBool active)
+  lang <- maybe (error "TODO: scim error") pure $ mapM parseLanguage $ Scim.preferredLanguage user
```

Comment:
How hard to go ahead and address this TODO in this PR?

Reply:
yes definitely, that's just a hole to be filled before the PR is merged.  i suppose the PR should not be in "ready for review" state yet...  :)